docs/release-notes/8.12.asciidoc

```diff
@@ -2,15 +2,32 @@
 == 8.12
 
 [discrete]
-[[enhancements-8.12.1]]
-==== Enhancements
+[[release-notes-8.12.2]]
+=== 8.12.2
 
-* Provides performance improvements related to image load and registry write events ({pull}175486[#175486]).
+[discrete]
+[[bug-fixes-8.12.2]]
+==== Bug fixes
+* Fixes long-running queries in Timeline and Events tables within Explore pages ({pull}176838[#176838]).
+* Updates the default {bedrock} connector API URL ({pull}176090[#176090]).
+* Ensures the risk score query only searches through alerts associated with the current user ({pull}175903[#175903]).
+* Fixes a bug that prevented numbers from being used for Osquery pack IDs ({pull}176507[#176507]).
+* Fixes bugs that affected the rule details page layout if rule filters were extremely long or had aliases ({pull}176590[#176590]).
```

Comment:
Not sure how much detail should go into this but these are actually two different bugs:
`Fixes bug that affected the rule details page layout if rule filters were extremely long`
`Fixes bug that affected rule filters in rule details page if the filters had aliases`

Reply:
++ These are two different bugs fixed by a single PR. The second one, related to filter aliases, had nothing to do with the page's layout.

Also FWIW "aliases" are called "Custom labels" in the UI.

Reply:
Thanks, both! How about this:

```suggestion
* Fixes a bug that affected the rule details page layout if rule filters were extremely long. Also fixes a bug that incorrectly caused rule filters to display instead of custom labels ({pull}176590[#176590]).
```

Reply:
I'd suggest:

"Also fixes a bug that incorrectly caused rule filters to display instead of **their** custom labels"

The logic is: if the rule filter has a custom label, display that label instead of the rule filter itself. The bug was showing the filter always, even for filters that had custom labels.

Hope that helps.